metadash/models/collections/attribute_mapped_list_collection.py

```diff
@@ -180,6 +194,24 @@ def _convert(self, target):
             for item in collection_or_item:
                 yield item
 
+class CreatorFactory:
+    """
+    If value is a list, create a instance for each element in value list.
+
+    If duplicate value in the list, get rid of it, it's supposed to have
+    unique value.
+    """
+
+    def __init__(self, owning_cl):
+        self.owning_cl = owning_cl
```

Comment:
Why not record owner collection itself, so no need to refresh the owenering class in __get__ of that collection class